lib/generators/shopify_app/shop_model/templates/shops.yml

```diff
@@ -0,0 +1,3 @@
+regular_shop:
+  shopify_domain: 'regular_shop.myshopify.com'
```

Comment:
Maybe make this `regular-shop.myshopify.com` since subdomains can't include underscores.